core/src/main/scala/com/yahoo/maha/core/query/druid/DruidQueryGenerator.scala

```diff
@@ -1002,8 +1002,8 @@ class DruidQueryGenerator(queryOptimizer: DruidQueryOptimizer
       }
     }
 
-    def getThetaSketchAggregatorFactory(outputFieldName: String, inputFieldName: String): AggregatorFactory = {
-      new SketchMergeAggregatorFactory(outputFieldName, inputFieldName, null, null, false, null)
+    def getThetaSketchAggregatorFactory(outputFieldName: String, inputFieldName: String, size: Int = 16384): AggregatorFactory = {
```

Comment:
same as above

Reply:
16384 is what these druid queries generate when no value is specified.